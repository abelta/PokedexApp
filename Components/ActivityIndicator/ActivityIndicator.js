import React from 'react';
import { ActivityIndicator } from 'react-native';

const ActivityIndicatorCustom = props => (
  <ActivityIndicator size="large" color="lightgrey" {...props} />
);

export default ActivityIndicatorCustom;
