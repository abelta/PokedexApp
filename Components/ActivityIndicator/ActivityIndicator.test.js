import React from 'react';
import renderer from 'react-test-renderer';
import ActivityIndicator from '.';

describe('ActivityIndicator', () => {
  it('should render properly', () => {
    const tree = renderer.create(<ActivityIndicator />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
