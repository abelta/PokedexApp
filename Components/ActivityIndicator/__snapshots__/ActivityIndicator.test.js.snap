// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActivityIndicator should render properly 1`] = `
<ActivityIndicator
  animating={true}
  color="lightgrey"
  hidesWhenStopped={true}
  size="large"
/>
`;
